[Crash] Friends&Foes (Forge)

### Bug Description

Crash with Friends&Foes mod

Forge 43.2.6
Friends&Foes mc1.19.2-1.6.8 (Forge)

Only mods present are debugify, clothconfig and friends&foes

### How to Reproduce?

Load minecraft with debugify and friend&foes mod

### Expected Behavior

_No response_

### Version

1.0.0

### Mod Loader Version

1.19.2

### Mod Loader

Forge

### Logs or additional context

```
MultiMC version: 0.7.0-3714


Launched instance in online mode

authserver.mojang.com resolves to:
    [18.67.28.46, 18.67.28.55, 18.67.28.38, 18.67.28.47]

session.minecraft.net resolves to:
    [100.24.172.95, 18.214.12.3, 34.197.32.8]

textures.minecraft.net resolves to:
    [13.107.237.35, 13.107.238.35, 2620:1ec:4e:1::35, 2620:1ec:4f:1::35]

api.mojang.com resolves to:
    [18.67.39.22, 18.67.39.54, 18.67.39.61, 18.67.39.127]


Minecraft folder is:
G:/Minecraft/MultiMC/instances/Age of Noobs/.minecraft


Java path is:
C:/Program Files/RedHat/java-17-openjdk-17.0.6.0.10-1/bin/javaw.exe


Checking Java version...
Java is version 17.0.6, using 64-bit architecture, from Red Hat, Inc..


Main Class:
  io.github.zekerzhayard.forgewrapper.installer.Main

Native path:
  G:/Minecraft/MultiMC/instances/Age of Noobs/natives

Traits:
traits FirstThreadOnMacOS
traits XR:Initial

Libraries:
  G:/Minecraft/MultiMC/libraries/org/lwjgl/lwjgl-glfw/3.3.1/lwjgl-glfw-3.3.1.jar
  G:/Minecraft/MultiMC/libraries/org/lwjgl/lwjgl-jemalloc/3.3.1/lwjgl-jemalloc-3.3.1.jar
  G:/Minecraft/MultiMC/libraries/org/lwjgl/lwjgl-openal/3.3.1/lwjgl-openal-3.3.1.jar
  G:/Minecraft/MultiMC/libraries/org/lwjgl/lwjgl-opengl/3.3.1/lwjgl-opengl-3.3.1.jar
  G:/Minecraft/MultiMC/libraries/org/lwjgl/lwjgl-stb/3.3.1/lwjgl-stb-3.3.1.jar
  G:/Minecraft/MultiMC/libraries/org/lwjgl/lwjgl-tinyfd/3.3.1/lwjgl-tinyfd-3.3.1.jar
  G:/Minecraft/MultiMC/libraries/org/lwjgl/lwjgl/3.3.1/lwjgl-3.3.1.jar
  G:/Minecraft/MultiMC/libraries/com/mojang/logging/1.0.0/logging-1.0.0.jar
  G:/Minecraft/MultiMC/libraries/com/mojang/blocklist/1.0.10/blocklist-1.0.10.jar
  G:/Minecraft/MultiMC/libraries/com/mojang/patchy/2.2.10/patchy-2.2.10.jar
  G:/Minecraft/MultiMC/libraries/com/github/oshi/oshi-core/5.8.5/oshi-core-5.8.5.jar
  G:/Minecraft/MultiMC/libraries/net/java/dev/jna/jna/5.10.0/jna-5.10.0.jar
  G:/Minecraft/MultiMC/libraries/net/java/dev/jna/jna-platform/5.10.0/jna-platform-5.10.0.jar
  G:/Minecraft/MultiMC/libraries/org/slf4j/slf4j-api/1.8.0-beta4/slf4j-api-1.8.0-beta4.jar
  G:/Minecraft/MultiMC/libraries/org/apache/logging/log4j/log4j-slf4j18-impl/2.17.0/log4j-slf4j18-impl-2.17.0.jar
  G:/Minecraft/MultiMC/libraries/com/ibm/icu/icu4j/70.1/icu4j-70.1.jar
  G:/Minecraft/MultiMC/libraries/com/mojang/javabridge/1.2.24/javabridge-1.2.24.jar
  G:/Minecraft/MultiMC/libraries/net/sf/jopt-simple/jopt-simple/5.0.4/jopt-simple-5.0.4.jar
  G:/Minecraft/MultiMC/libraries/io/netty/netty-common/4.1.77.Final/netty-common-4.1.77.Final.jar
  G:/Minecraft/MultiMC/libraries/io/netty/netty-buffer/4.1.77.Final/netty-buffer-4.1.77.Final.jar
  G:/Minecraft/MultiMC/libraries/io/netty/netty-codec/4.1.77.Final/netty-codec-4.1.77.Final.jar
  G:/Minecraft/MultiMC/libraries/io/netty/netty-handler/4.1.77.Final/netty-handler-4.1.77.Final.jar
  G:/Minecraft/MultiMC/libraries/io/netty/netty-resolver/4.1.77.Final/netty-resolver-4.1.77.Final.jar
  G:/Minecraft/MultiMC/libraries/io/netty/netty-transport/4.1.77.Final/netty-transport-4.1.77.Final.jar
  G:/Minecraft/MultiMC/libraries/io/netty/netty-transport-native-unix-common/4.1.77.Final/netty-transport-native-unix-common-4.1.77.Final.jar
  G:/Minecraft/MultiMC/libraries/io/netty/netty-transport-classes-epoll/4.1.77.Final/netty-transport-classes-epoll-4.1.77.Final.jar
  G:/Minecraft/MultiMC/libraries/com/google/guava/failureaccess/1.0.1/failureaccess-1.0.1.jar
  G:/Minecraft/MultiMC/libraries/com/google/guava/guava/31.0.1-jre/guava-31.0.1-jre.jar
  G:/Minecraft/MultiMC/libraries/org/apache/commons/commons-lang3/3.12.0/commons-lang3-3.12.0.jar
  G:/Minecraft/MultiMC/libraries/commons-io/commons-io/2.11.0/commons-io-2.11.0.jar
  G:/Minecraft/MultiMC/libraries/commons-codec/commons-codec/1.15/commons-codec-1.15.jar
  G:/Minecraft/MultiMC/libraries/com/mojang/brigadier/1.0.18/brigadier-1.0.18.jar
  G:/Minecraft/MultiMC/libraries/com/mojang/datafixerupper/5.0.28/datafixerupper-5.0.28.jar
  G:/Minecraft/MultiMC/libraries/com/google/code/gson/gson/2.8.9/gson-2.8.9.jar
  G:/Minecraft/MultiMC/libraries/com/mojang/authlib/3.11.49/authlib-3.11.49.jar
  G:/Minecraft/MultiMC/libraries/org/apache/commons/commons-compress/1.21/commons-compress-1.21.jar
  G:/Minecraft/MultiMC/libraries/org/apache/httpcomponents/httpclient/4.5.13/httpclient-4.5.13.jar
  G:/Minecraft/MultiMC/libraries/commons-logging/commons-logging/1.2/commons-logging-1.2.jar
  G:/Minecraft/MultiMC/libraries/org/apache/httpcomponents/httpcore/4.4.14/httpcore-4.4.14.jar
  G:/Minecraft/MultiMC/libraries/it/unimi/dsi/fastutil/8.5.6/fastutil-8.5.6.jar
  G:/Minecraft/MultiMC/libraries/org/apache/logging/log4j/log4j-api/2.17.0/log4j-api-2.17.0.jar
  G:/Minecraft/MultiMC/libraries/org/apache/logging/log4j/log4j-core/2.17.0/log4j-core-2.17.0.jar
  G:/Minecraft/MultiMC/libraries/com/mojang/text2speech/1.13.9/text2speech-1.13.9.jar
  G:/Minecraft/MultiMC/libraries/io/github/zekerzhayard/ForgeWrapper/mmc3/ForgeWrapper-mmc3.jar
  G:/Minecraft/MultiMC/libraries/cpw/mods/securejarhandler/2.1.4/securejarhandler-2.1.4.jar
  G:/Minecraft/MultiMC/libraries/org/ow2/asm/asm/9.3/asm-9.3.jar
  G:/Minecraft/MultiMC/libraries/org/ow2/asm/asm-commons/9.3/asm-commons-9.3.jar
  G:/Minecraft/MultiMC/libraries/org/ow2/asm/asm-tree/9.3/asm-tree-9.3.jar
  G:/Minecraft/MultiMC/libraries/org/ow2/asm/asm-util/9.3/asm-util-9.3.jar
  G:/Minecraft/MultiMC/libraries/org/ow2/asm/asm-analysis/9.3/asm-analysis-9.3.jar
  G:/Minecraft/MultiMC/libraries/net/minecraftforge/accesstransformers/8.0.4/accesstransformers-8.0.4.jar
  G:/Minecraft/MultiMC/libraries/org/antlr/antlr4-runtime/4.9.1/antlr4-runtime-4.9.1.jar
  G:/Minecraft/MultiMC/libraries/net/minecraftforge/eventbus/6.0.3/eventbus-6.0.3.jar
  G:/Minecraft/MultiMC/libraries/net/minecraftforge/forgespi/6.0.0/forgespi-6.0.0.jar
  G:/Minecraft/MultiMC/libraries/net/minecraftforge/coremods/5.0.1/coremods-5.0.1.jar
  G:/Minecraft/MultiMC/libraries/cpw/mods/modlauncher/10.0.8/modlauncher-10.0.8.jar
  G:/Minecraft/MultiMC/libraries/net/minecraftforge/unsafe/0.2.0/unsafe-0.2.0.jar
  G:/Minecraft/MultiMC/libraries/com/electronwill/night-config/core/3.6.4/core-3.6.4.jar
  G:/Minecraft/MultiMC/libraries/com/electronwill/night-config/toml/3.6.4/toml-3.6.4.jar
  G:/Minecraft/MultiMC/libraries/org/apache/maven/maven-artifact/3.8.5/maven-artifact-3.8.5.jar
  G:/Minecraft/MultiMC/libraries/net/jodah/typetools/0.8.3/typetools-0.8.3.jar
  G:/Minecraft/MultiMC/libraries/net/minecrell/terminalconsoleappender/1.2.0/terminalconsoleappender-1.2.0.jar
  G:/Minecraft/MultiMC/libraries/org/jline/jline-reader/3.12.1/jline-reader-3.12.1.jar
  G:/Minecraft/MultiMC/libraries/org/jline/jline-terminal/3.12.1/jline-terminal-3.12.1.jar
  G:/Minecraft/MultiMC/libraries/org/spongepowered/mixin/0.8.5/mixin-0.8.5.jar
  G:/Minecraft/MultiMC/libraries/org/openjdk/nashorn/nashorn-core/15.3/nashorn-core-15.3.jar
  G:/Minecraft/MultiMC/libraries/net/minecraftforge/JarJarSelector/0.3.16/JarJarSelector-0.3.16.jar
  G:/Minecraft/MultiMC/libraries/net/minecraftforge/JarJarMetadata/0.3.16/JarJarMetadata-0.3.16.jar
  G:/Minecraft/MultiMC/libraries/cpw/mods/bootstraplauncher/1.1.2/bootstraplauncher-1.1.2.jar
  G:/Minecraft/MultiMC/libraries/net/minecraftforge/JarJarFileSystems/0.3.16/JarJarFileSystems-0.3.16.jar
  G:/Minecraft/MultiMC/libraries/net/minecraftforge/fmlloader/1.19.2-43.2.6/fmlloader-1.19.2-43.2.6.jar
  G:/Minecraft/MultiMC/libraries/com/mojang/minecraft/1.19.2/minecraft-1.19.2-client.jar

Native libraries:
  G:/Minecraft/MultiMC/libraries/org/lwjgl/lwjgl-glfw/3.3.1/lwjgl-glfw-3.3.1-natives-windows.jar
  G:/Minecraft/MultiMC/libraries/org/lwjgl/lwjgl-jemalloc/3.3.1/lwjgl-jemalloc-3.3.1-natives-windows.jar
  G:/Minecraft/MultiMC/libraries/org/lwjgl/lwjgl-openal/3.3.1/lwjgl-openal-3.3.1-natives-windows.jar
  G:/Minecraft/MultiMC/libraries/org/lwjgl/lwjgl-opengl/3.3.1/lwjgl-opengl-3.3.1-natives-windows.jar
  G:/Minecraft/MultiMC/libraries/org/lwjgl/lwjgl-stb/3.3.1/lwjgl-stb-3.3.1-natives-windows.jar
  G:/Minecraft/MultiMC/libraries/org/lwjgl/lwjgl-tinyfd/3.3.1/lwjgl-tinyfd-3.3.1-natives-windows.jar
  G:/Minecraft/MultiMC/libraries/org/lwjgl/lwjgl/3.3.1/lwjgl-3.3.1-natives-windows.jar

Mods:
  [✔️] cloth-config-8.2.88-forge.jar
  [✔️] debugify_reforged-1.0.0-forge-1.19.2.jar
  [✔️] friendsandfoes-forge-mc1.19.2-1.6.8.jar

Params:
  --username  --version 1.19.2 --gameDir G:/Minecraft/MultiMC/instances/Age of Noobs/.minecraft --assetsDir G:/Minecraft/MultiMC/assets --assetIndex 1.19 --uuid  --accessToken  --userType  --versionType release --launchTarget forgeclient --fml.forgeVersion 43.2.6 --fml.mcVersion 1.19.2 --fml.forgeGroup net.minecraftforge --fml.mcpVersion 20220805.130853

Window size: 1024 x 512

Java Arguments:
[-XX:+UseG1GC, -Dsun.rmi.dgc.server.gcInterval=2147483646, -XX:+UnlockExperimentalVMOptions, -XX:G1NewSizePercent=20, -XX:G1ReservePercent=20, -XX:MaxGCPauseMillis=51, -XX:G1HeapRegionSize=32M, -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump, -Xms1024m, -Xmx16384m, -Duser.language=en]


Minecraft process ID: 15460


Using onesix launcher.

Checking: MC_SLIM
Checking: MERGED_MAPPINGS
Checking: MAPPINGS
Checking: MC_EXTRA
Checking: MOJMAPS
Checking: PATCHED
Checking: MC_SRG
2023-02-28 00:05:36,461 main WARN Advanced terminal features are not available in this environment
[00:05:36] [main/INFO] [cp.mo.mo.Launcher/MODLAUNCHER]: ModLauncher running: args [--username, Aethamus, --version, 1.19.2, --gameDir, G:/Minecraft/MultiMC/instances/Age of Noobs/.minecraft, --assetsDir, G:/Minecraft/MultiMC/assets, --assetIndex, 1.19, --uuid, <PROFILE ID>, --accessToken, ????????, --userType, msa, --versionType, release, --launchTarget, forgeclient, --fml.forgeVersion, 43.2.6, --fml.mcVersion, 1.19.2, --fml.forgeGroup, net.minecraftforge, --fml.mcpVersion, 20220805.130853, --width, 1024, --height, 512]
[00:05:36] [main/INFO] [cp.mo.mo.Launcher/MODLAUNCHER]: ModLauncher 10.0.8+10.0.8+main.0ef7e830 starting: java version 17.0.6 by Red Hat, Inc.; OS Windows 10 arch amd64 version 10.0
[00:05:36] [main/INFO] [mixin/]: SpongePowered MIXIN Subsystem Version=0.8.5 Source=union:/G:/Minecraft/MultiMC/libraries/org/spongepowered/mixin/0.8.5/mixin-0.8.5.jar%23118!/ Service=ModLauncher Env=CLIENT
[00:05:37] [main/WARN] [ne.mi.fm.lo.mo.ModFileParser/LOADING]: Mod file G:\Minecraft\MultiMC\libraries\net\minecraftforge\fmlcore\1.19.2-43.2.6\fmlcore-1.19.2-43.2.6.jar is missing mods.toml file
[00:05:37] [main/WARN] [ne.mi.fm.lo.mo.ModFileParser/LOADING]: Mod file G:\Minecraft\MultiMC\libraries\net\minecraftforge\javafmllanguage\1.19.2-43.2.6\javafmllanguage-1.19.2-43.2.6.jar is missing mods.toml file
[00:05:37] [main/WARN] [ne.mi.fm.lo.mo.ModFileParser/LOADING]: Mod file G:\Minecraft\MultiMC\libraries\net\minecraftforge\lowcodelanguage\1.19.2-43.2.6\lowcodelanguage-1.19.2-43.2.6.jar is missing mods.toml file
[00:05:37] [main/WARN] [ne.mi.fm.lo.mo.ModFileParser/LOADING]: Mod file G:\Minecraft\MultiMC\libraries\net\minecraftforge\mclanguage\1.19.2-43.2.6\mclanguage-1.19.2-43.2.6.jar is missing mods.toml file
[00:05:37] [main/INFO] [ne.mi.fm.lo.mo.JarInJarDependencyLocator/]: Found 1 dependencies adding them to mods collection
Failed to start Minecraft:
java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at io.github.zekerzhayard.forgewrapper.installer.Main.main(Main.java:57)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:214)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:249)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)
Caused by: java.lang.module.ResolutionException: Modules MixinExtras and debugify export package com.llamalad7.mixinextras to module minecraft
	at java.base/java.lang.module.Resolver.resolveFail(Resolver.java:901)
	at java.base/java.lang.module.Resolver.failTwoSuppliers(Resolver.java:815)
	at java.base/java.lang.module.Resolver.checkExportSuppliers(Resolver.java:736)
	at java.base/java.lang.module.Resolver.finish(Resolver.java:380)
	at java.base/java.lang.module.Configuration.<init>(Configuration.java:140)
	at java.base/java.lang.module.Configuration.resolveAndBind(Configuration.java:494)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.ModuleLayerHandler.buildLayer(ModuleLayerHandler.java:75)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.TransformationServicesHandler.buildTransformingClassLoader(TransformationServicesHandler.java:60)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.Launcher.run(Launcher.java:104)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.Launcher.main(Launcher.java:77)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23)
	at cpw.mods.bootstraplauncher@1.1.2/cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141)
	... 13 more
Exiting with -1
Process exited with code -1 (0xffffffffffffffff).
Please note that usually neither the exit code, nor its description are enough to diagnose issues!
Always upload the entire log and not just the exit code.

```

### Has not been reported.

- [X] I made sure this bug hasn't already been reported.

### Is on most update to date version.

- [X] I made sure I am using the most up to date mod loader and mod version.

--- a/forge/build.gradle.kts
+++ b/forge/build.gradle.kts
@@ -82,6 +82,8 @@ tasks {
 
         configurations = listOf(shadowCommon)
         archiveClassifier.set("dev-shadow")
+        
+        mergeServiceFiles()
     }
 
     remapJar {

